### ISSUE TITLE
Problem with recurring payments

### ISSUE THREAD

**Issue by lizmestres** [drastik/com.drastikbydesign.stripe#181]
I'm using Drupal 7.53 and civicrm 7.x-4.7.14. I just set up stripe, but recurring payments continually display "pending." I set the webhook path in my stripe account and the charges are taking place. What am I missing?

**Comment by h-c-c** [comment]
Are you on the latest GitHub 4.7-dev version?

**Comment by lizmestres** [comment]
I'm on 4.7.1

**Comment by h-c-c** [comment]
Please upgrade to the latest GitHub 4.7-dev version.

**Comment by drastik** [comment]
@h-c-c You feel like we're ready for a new release at this point?

**Comment by lizmestres** [comment]
I've upgraded but nothing seems to have changed.

**Comment by h-c-c** [comment]
@lizmestres do you get 'access denied' when you go to:

http://yourdomain.org/civicrm/stripe/webhook 

@drastik it's close.  But , there's a issue I want to address upstream and then act on the outcome...  Plus New Feature:refunds is pretty close to being done. \o/  I need some questions answered first.

**Comment by lizmestres** [comment]
No, it takes me to the civicrm dashboard

**Comment by h-c-c** [comment]
Ok, that's a problem.  Logged in or not, you should get a json encode error.   

We have to go though some very basic troubleshooting steps.  Can you confirm your Stripe extension is enabled?  

http://yourdomain.org/civicrm/admin/extensions?reset=1&action=browse

**Comment by h-c-c** [comment]
And if it is, look in the Stripe extension directory com.drastikbydesign.stripe.    Do you see a subdirectory com.drastikbydesign.stripe?  

Or do you see:
```CRM
info.xml
js
LICENSE.txt
packages
README.md
stripe.civix.php
stripe.php
templates
xml
```

If the former, the files in the subdirectory need to come up a level.

**Comment by lizmestres** [comment]
I see the second, plus several other folders:
CRM (folder)
info.xml
js (folder)
LICENSE.txt
packages (folder)
README.md
stripe.civix.php
stripe.php
templates (folder)
xml (folder)

**Comment by lizmestres** [comment]
For this instruction:
"You will have to make a Webhook rule in your Stripe.com account and enter this path for recurring charges to end!, " I went to the webhooks section of my account and clicked "add endpoint" and then put the full path in. Is that what's meant by making a Webhook rule?

**Comment by drastik** [comment]
Yes.  The path is different between CMS's (Drupal, Wordpress, Joomla).

**Comment by drastik** [comment]
This is what @h-c-c means by the error message you *should* see when visiting that path:
![selection_139](https://cloud.githubusercontent.com/assets/1258227/22409079/efc636f0-e649-11e6-8c9a-211893086ce8.png)

**Comment by lizmestres** [comment]
This is what I put in my stripe acct for a webhook endpoint url:
https://globaljusticecenter.org/civicrm/webhook (it's drupal)

**Comment by lizmestres** [comment]
I just disabled and enabled the extension and got the message that the url should be https://site.org/civicrm/stripe/webhook -- I added the stripe part and now get the error message you mention. Hopefully, that was the problem.

**Comment by drastik** [comment]
That is the URL as it should be in your `stripe.com` settings also.

**Comment by lizmestres** [comment]
I think it's fine now--I misread the directions so many times. Thanks for all your help.

**Comment by h-c-c** [comment]
Sweet!  You're welcome!
